.github/workflows/build.yml

```
@@ -392,6 +356,14 @@ jobs:
     runs-on: ubuntu-22.04
     needs: build-docker
     steps:
+      - name: Login to Azure - PROD Subscription
+        uses: Azure/login@92a5484dfaf04ca78a94597f4f19fea633851fa2 # v1.4.7
+        with:
+          creds: ${{ secrets.AZURE_PROD_KV_CREDENTIALS }}
+
+      - name: Login to PROD ACR
+        run: az acr login -n $AZ_REGISTRY
```

```suggestion
        run: az acr login -n $AZ_REGISTRY --only-show-errors
```
🎨 The addition of the `--only-show-errors` allows you to reduce output about warnings regarding the addition of `.azurecr.io` in your registry name when using `acr login`.